### PR TITLE
data: JustFelix-Dev

### PR DESCRIPTION
**Pull request by JustFelix-Dev**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/JustFelix-Dev.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      "url": "https://twitter.com/justfelix_dev"
+    },
+     {
+      "icons": "FaLinkedin",
+      "url": "https://www.linkedin.com/in/justfelixowolabi/"
+    },
+     {
+      "icons": "FaGithub",
+      "url": "https://github.com/JustFelix-Dev"
+    },
```
Comment (Sindhuinti) [inline data/JustFelix-Dev.json:23]:
```suggestion
    }
```

**Review by Sindhuinti — CHANGES_REQUESTED** [review]
Please remove this comment ( not needed)

File: data/JustFelix-Dev.json (hunk 1 above)
Comment (JustFelix-Dev) [reply data/JustFelix-Dev.json]: Thank you for the review. Done!

**Review by sital002 — APPROVED** [review]
Looks good 👍
Welcome to Eddiehub community 🎉🎉

**Pull request merged**